lib/voxpupuli/test/rake.rb

```diff
@@ -12,15 +12,22 @@
 namespace :check do
   desc 'Check for trailing whitespace'
   task :trailing_whitespace do
+    errors = []
+
     Dir.glob('**/*.md', File::FNM_DOTMATCH).sort.each do |filename|
       next if filename =~ %r{^((modules|acceptance|\.?vendor|spec/fixtures|pkg)/|REFERENCE.md)}
       File.foreach(filename).each_with_index do |line, index|
         if line =~ %r{\s\n$}
-          puts "#{filename} has trailing whitespace on line #{index + 1}"
-          exit 1
+          errors << "#{filename} has trailing whitespace on line #{index + 1}"
         end
       end
     end
+
+    unless errors.empty?
```

Comment:
I'm still hesitant on using `unless` when I can avoid it
```suggestion
    if errors.any?
```